src/WebJobs.Extensions.ApiHub/Config/ApiHubConfiguration.cs

```diff
@@ -86,7 +88,7 @@ string IFileTriggerStrategy<ApiHubFile>.GetPath(ApiHubFile file)
         {
             if (file == null)
             {
-                throw new ArgumentNullException("file");
+                throw new ArgumentNullException(nameof(file));
```

Comment:
Unfortunately use of new C# features will break the CI - the requisite version of msbuild is not yet supported on our TeamCity instance :(